### PR TITLE
AC Test fix - LRAC-11392 | master

### PR DESCRIPTION
**Pull request by joshitagaki**
https://issues.liferay.com/browse/LRAC-11392

### CODE REVIEW

**Comment by joshitagaki** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSites.macro [hunk 1]
```diff
@@ -1,5 +1,15 @@
 definition {
 
+	macro getCurrentMetric {
+		WaitForVisible(
+			key_cardTitle = "Visitors Behavior",
+			locator1 = "ACCards#CARD_TITLE");
+
+		var current = selenium.getText("//span[@class='title'and text()='${metricName}']/..//div[@class='metric-value']");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSites.macro:8]: Hey @joshitagaki, very nice work here. I think you could create a path to not use the path directly in the `.macro`. What do you think?

Comment (joshitagaki) [reply portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSites.macro]: Thanks! Yea I moved it to .path file using an unused path locator

**Review by ViniLopes20 — APPROVED** [review]
LGTM, thanks Josh 🎉

**Review by calvinctvo — APPROVED** [review]
Great work @joshitagaki ! LGTM

**Review by christianstokes — APPROVED** [review]
Looks good, thank you @joshitagaki!

**Comment by calvinctvo** [comment]
ci:forward

**Pull request closed without merging**